Sources/Rendering/Core/Mapper/index.js

```diff
@@ -185,117 +441,7 @@ function vtkMapper(publicAPI, model) {
-      // result in incorrect images when the texture value should be
-      // above or below 1000, but I don't have a better solution.
-      if (texCoordS > 1000.0) {
-        texCoordS = 1000.0;
-      } else if (texCoordS < -1000.0) {
-        texCoordS = -1000.0;
-      }
-    }
-    return { texCoordS, texCoordT };
-  };
-
-  //-----------------------------------------------------------------------------
-  publicAPI.createColorTextureCoordinates = (
-    input,
-    component,
-    range,
-    tableNumberOfColors,
-    useLogScale
-  ) => {
-    // We have to change the range used for computing texture
-    // coordinates slightly to accommodate the special above- and
-    // below-range colors that are the first and last texels,
-    // respectively.
-    const scalarTexelWidth = (range[1] - range[0]) / tableNumberOfColors;
-
-    const paddedRange = [
-      range[0] - scalarTexelWidth,
-      range[1] + scalarTexelWidth,
-    ];
-    const invRangeWidth = 1.0 / (paddedRange[1] - paddedRange[0]);
-
-    const inputV = input.getData();
-    const numScalars = input.getNumberOfTuples();
-    const numComps = input.getNumberOfComponents();
-    const output = vtkDataArray.newInstance({
-      numberOfComponents: 2,
-      values: new Float32Array(numScalars * 2),
-    });
-    const outputV = output.getData();
-
-    if (component < 0 || component >= numComps) {
-      // Convert the magnitude of all components to texture coordinates
-      let inputIdx = 0;
-      let outputIdx = 0;
-      for (let scalarIdx = 0; scalarIdx < numScalars; ++scalarIdx) {
-        let sum = 0;
-        for (let compIdx = 0; compIdx < numComps; ++compIdx) {
-          sum += inputV[inputIdx] * inputV[inputIdx];
-          inputIdx++;
-        }
-        let magnitude = Math.sqrt(sum);
-        if (useLogScale) {
-          magnitude = vtkLookupTable.applyLogScale(magnitude, range, range);
-        }
-        const outputs = publicAPI.scalarToTextureCoordinate(
-          magnitude,
-          paddedRange[0],
-          invRangeWidth
-        );
-        outputV[outputIdx++] = outputs.texCoordS;
-        outputV[outputIdx++] = outputs.texCoordT;
-      }
-    } else {
-      // Convert one of the components to texture coordinates
-      let inputIdx = component;
-      let outputIdx = 0;
-      for (let scalarIdx = 0; scalarIdx < numScalars; ++scalarIdx) {
-        let inputValue = inputV[inputIdx];
-        if (useLogScale) {
-          inputValue = vtkLookupTable.applyLogScale(inputValue, range, range);
-        }
-        const outputs = publicAPI.scalarToTextureCoordinate(
-          inputValue,
-          paddedRange[0],
-          invRangeWidth
-        );
-        outputV[outputIdx++] = outputs.texCoordS;
-        outputV[outputIdx++] = outputs.texCoordT;
-        inputIdx += numComps;
-      }
-    }
-    return output;
-  };
-
-  publicAPI.mapScalarsToTexture = (scalars, alpha) => {
+  publicAPI._mapScalarsToTexture = (scalars, cellFlag, alpha) => {
```

Comment:
this is not removing it from the public API, to make the method protected, you need to add it to "model" instead of "publicAPI"